package.json

```diff
@@ -8,6 +8,8 @@
   "dependencies": {
     "bootstrap": "^3.3.4",
     "crypto-js": "^3.1.2-5",
+    "es5-shim": "^4.1.7",
```

Comment:
Not sure I understand why es5-shim is necessary ... I thought Sentry targeted ES5-compatible browsers.